### DISCUSSION TITLE
ElasticSearch Showing Mapping Error. I had created Index and Aliase and done the mapping and hit the search API it showing this error [knowledge-Platform version 5.1.0]

### DISCUSSION THREAD

**Discussion by Tusharmahajan12** [Sunbird-Knowlg/Community#94] [Q&A] [answered]
`  "errmsg": "method [POST], host [http://localhost:9200], URI [/compositesearch/_search?rest_total_hits_as_int=true&typed_keys=true&ignore_unavailable=false&expand_wildcards=open&allow_no_indices=true&ignore_throttled=true&search_type=query_then_fetch&batched_reduce_size=512], status line [HTTP/1.1 400 Bad Request]\n{\"error\":{\"root_cause\":[{\"type\":\"query_shard_exception\",\"reason\":\"No mapping found for [name.raw] in order to sort on\",\"index_uuid\":\"6LEHuAj1Sy24oL_ue_sthg\",\"index\":\"compositesearch_v1\"}],\"type\":\"search_phase_execution_exception\",\"reason\":\"all shards failed\",\"phase\":\"query\",\"grouped\":true,\"failed_shards\":[{\"shard\":0,\"index\":\"compositesearch_v1\",\"node\":\"1s-LwausRmOWrsdo2UrByw\",\"reason\":{\"type\":\"query_shard_exception\",\"reason\":\"No mapping found for [name.raw] in order to sort on\",\"index_uuid\":\"6LEHuAj1Sy24oL_ue_sthg\",\"index\":\"compositesearch_v1\"}}]},\"status\":400}"`

**Comment by Krishnaj20** [comment, marked as answer]
@maheshkumargangula - Pls check and respond to this in Kartheek's absence.

@pallakartheekreddy

**Reply by maheshkumargangula** [reply]
@Tusharmahajan12 – Based on the above error, there is no mapping available for the `name` property to `compositesearch` index. Please check the index has mapping configured properly or not in ES. Moreover, try to ensure there are few documents in the index. 
```No mapping found for [name.raw] in order to sort```
Cc: @Krishnaj20 @pallakartheekreddy

**Reply by Tusharmahajan12** [reply]
I had done that still have the issue

**Comment by pallakartheekreddy** [comment]
@Tusharmahajan12 Is your query resolved? If so, pls mark it as answered

**Reply by Tusharmahajan12** [reply]
not solved but i will mark as answered
